README.md

```diff
@@ -109,13 +109,27 @@ You probably need to set some environment variables for the client to be useful.
 
 ## Troubleshooting
 
+### I'm getting compilation errors
+
+Make sure you are using NodeJS 12. This may involve installing an LTS version from a package manager rather than the latest stable version.
```

Comment:
Ah, this is easily avoidable - it's just because the version of one of the usb dependencies is too old to work with latest node. I should just update the packages so this will be fixed.